cs/cli/vw_label.h

```diff
@@ -70,9 +70,14 @@ public ref class ContextualBanditLabel sealed : ILabel
 
     void set(float value)
     { if (value < 0 || value >1)
-        throw gcnew ArgumentOutOfRangeException("invalid probability: " + value);
-
-      m_probability = value;
+      {
+        if (value > 1 && value < 1.00001)
```

Comment:
Is 0.00001 documented anywhere as our error margin?
Shouldn't it be a constant to make it consistent across label value operations?

Reply:
Yes, I really like to add a constant somewhere. Where is the right place to do that?

Reply:
There is a constant.h file - maybe that is the right place?

Reply:
![image](https://user-images.githubusercontent.com/13020512/59536219-f1dd3700-8ea7-11e9-9abb-d862c98ca69e.png)
Why do we have some many constant.h files? Shouldn't they all be combined? If not, then which one is the "best" constant.h for me to use?